docusaurus/docs/protocol/primitives/claim_and_proof_lifecycle.md

```diff
@@ -126,6 +128,17 @@ gantt
 
 - [`poktroll.shared.Params` / `sharedtypes.Params`](https://github.com/pokt-network/poktroll/blob/main/proto/poktroll/shared/params.proto)
 
+### Claim Expiration
+
+If a claim requires a proof (as determined by [Probabilistic Proofs](probabilistic_proofs.md)) and a `Supplier` fails
+to submit a `Proof` before the `Proof Window` closes, the claim will expire and the `Supplier` will forfeit any
+rewards for the work done.
+
+Claims MUST expire (and therefore the proof window MUST close) for the following reasons:
+
+1. The mint & burn associated with a given claim's settlement MUST occur while the application stake is still locked and applications must be allowed complete unstaking in finite time.
+1. Claim settlement SHOULD be limited to considering claims created within a rolling window of blocks to decouple settlement from a long-tail accumulation of unsettled claims.
+1. Proofs MUST be pruned to prevent network state bloat over time. Pruning proofs makes the number of proofs in network state at any given time a function of recent relay demand.
```

Comment:
```suggestion
1. Proofs MUST be pruned to prevent network state bloat over time. Pruning proofs makes the number of proofs in network state at any given time a function of recent relay demand.

```